tools/harness-thci/OpenThread.py

```diff
@@ -1718,7 +1718,7 @@ def configBorderRouter(
         if TESTHARNESS_VERSION == TESTHARNESS_1_2 and P_dp == 0 and not self.__isOpenThreadRunning():
             self.__useDefaultDomainPrefix = False
 
-        if TESTHARNESS_VERSION == TESTHARNESS_1_2 and self.IsBackboneRouter:
+        if TESTHARNESS_VERSION == TESTHARNESS_1_2 and self.IsBorderRouter:
```

Comment:
Using `IsBorderRouter` is even more incorrect because it will fail BR devices in 1.1 certification (Border Router role).

Reply:
I see. In that case probably it's better to focus efforts in https://threadgroup.atlassian.net/browse/DEV-2195

Reply:
agree.